### PR TITLE
[Merged by Bors] - feat : Stacks: Lemma 0052 (3) irreducible component of noetherian space contains a nonempty open subset

### PR DESCRIPTION
**Pull request by jjaassoonn**
any irreducible components of a noetherian space contains a nonempty open subset 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/NoetherianSpace.lean [hunk 1]
```diff
@@ -77,6 +77,7 @@ protected theorem _root_.Inducing.noetherianSpace [NoetherianSpace α] {i : β 
   (noetherianSpace_iff_opens _).2 fun _ => hi.isCompact_iff.1 (NoetherianSpace.isCompact _)
 #align topological_space.inducing.noetherian_space Inducing.noetherianSpace
 
+/-- [Stacks: 0052 (1) ](https://stacks.math.columbia.edu/tag/0052)-/
```
Comment (jcommelin) [inline Mathlib/Topology/NoetherianSpace.lean:80]:
```suggestion
/-- [Stacks: Lemma 0052 (1)](https://stacks.math.columbia.edu/tag/0052)-/
```

File: Mathlib/Topology/NoetherianSpace.lean [hunk 2]
```diff
@@ -217,4 +219,50 @@ theorem NoetherianSpace.finite_irreducibleComponents [NoetherianSpace α] :
   rwa [ht.antisymm (hs.2 (hSi _ htS) ht)]
 #align topological_space.noetherian_space.finite_irreducible_components TopologicalSpace.NoetherianSpace.finite_irreducibleComponents
 
+/-- [Stacks: Lemma 0052 (3)](https://stacks.math.columbia.edu/tag/0052) -/
+theorem NoetherianSpace.exists_open_ne_empty_le_irreducibleComponents [NoetherianSpace α]
```
Comment (jcommelin) [inline Mathlib/Topology/NoetherianSpace.lean:223]:
```suggestion
theorem NoetherianSpace.exists_open_ne_empty_le_irreducibleComponent [NoetherianSpace α]
```

**Review by jcommelin — APPROVED** [review]
bors d+

**Comment by ChrisHughes24** [comment]
Looks like this is the one with the problem

bors r-

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**